docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "",
+      "poc_github": "",
```

Comment:
please add your (or whoever will be responsible for keeping this up-to-date) github username here

Reply:
Done